### PR TITLE
Moderator pull request

### PR DESCRIPTION
**Pull request by RomanVantukh**

### CODE REVIEW

File: eQuiz/modules/eQuiz.Web/Areas/Moderator/Controllers/QuizController.cs [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace eQuiz.Web.Areas.Moderator.Controllers
+{
+    public class QuizController : Controller
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Areas/Moderator/Controllers/QuizController.cs]: The controller should be inherited from BaseController.

File: eQuiz/modules/eQuiz.Web/Areas/Moderator/Controllers/QuizQuestionController.cs [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using System.Data.Entity;
+using Newtonsoft.Json;
+
+namespace eQuiz.Web.Areas.Moderator.Controllers
+{
+    public class QuizQuestionController : Controller
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Areas/Moderator/Controllers/QuizQuestionController.cs]: The controller should be inherited from BaseController.

File: eQuiz/modules/eQuiz.Web/Areas/Moderator/Controllers/UserGroupController.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+﻿using eQuiz.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace eQuiz.Web.Areas.Moderator.Controllers
+{
+    public class UserGroupController : Controller
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Areas/Moderator/Controllers/UserGroupController.cs]: The controller should be inherited from BaseController.

File: eQuiz/modules/eQuiz.Web/Areas/Moderator/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -0,0 +1,39 @@
+﻿
```
Comment (vvoityshyn) [inline eQuiz/modules/eQuiz.Web/Areas/Moderator/Views/Shared/_Layout.cshtml:1]: We should not have a separate layout page in this area.
Use Layout.cshtml from the common area instead.

File: eQuiz/modules/eQuiz.Web/Web.config [hunk 5]
```diff
@@ -4,41 +4,55 @@
   http://go.microsoft.com/fwlink/?LinkId=301880
   -->
 <configuration>
+  <configSections>
+    <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 -->
+    <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
```
Comment (voytyshyn) [inline eQuiz/modules/eQuiz.Web/Web.config]: We should not have Entity Framework configuration here.

**Comment by vvoityshyn** [comment]
The changes have been merged.
So closing the pull request...

**Pull request closed without merging**